lib/sparse/src/index/inverted_index/inverted_index_mmap.rs

```diff
@@ -42,8 +42,16 @@ struct PostingListFileHeader {
 }
 
 impl InvertedIndex for InvertedIndexMmap {
-    fn open(path: &Path) -> std::io::Result<Option<Self>> {
-        Self::load(path).map(Some)
+    fn open(path: &Path) -> std::io::Result<Self> {
+        Self::load(path)
+    }
+
+    fn save(&self, path: &Path) -> std::io::Result<()> {
```

Comment:
is `save` called on immutable segments?

Reply:
For immutable RAM we have to call save manually because we cannot do it while `build` function - we don't know is index mutable (without files) or immutable (with files)